perf/benchmark_radix_tree.cpp

```diff
@@ -0,0 +1,113 @@
+/*
+    Copyright (c) 2018 Contributors as noted in the AUTHORS file
+
+    This file is part of libzmq, the ZeroMQ core engine in C++.
+
+    libzmq is free software; you can redistribute it and/or modify it under
+    the terms of the GNU Lesser General Public License (LGPL) as published
+    by the Free Software Foundation; either version 3 of the License, or
+    (at your option) any later version.
+
+    As a special exception, the Contributors give you permission to link
+    this library with independent modules to produce an executable,
+    regardless of the license terms of these independent modules, and to
+    copy and distribute the resulting executable under terms of your choice,
+    provided that you also meet, for each linked independent module, the
+    terms and conditions of the license of that module. An independent
+    module is a module which is not derived from or based on this library.
+    If you modify this library, you must extend this exception to your
+    version of the library.
+
+    libzmq is distributed in the hope that it will be useful, but WITHOUT
+    ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+    FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public
+    License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <trie.hpp>
+#include <radix_tree.hpp>
```

Comment:
both of these are not system includes, they need to be quoted:

```
#include "trie.hpp"
#include "radix_tree.hpp"
```

otherwise the build fails

Reply:
I tried this but it fails with the same error. It looks like a dependency issue; the `src/libzmq.la` target builds `libzmq.a`, so adding this line seems to fix it:

```
perf_benchmark_radix_tree_DEPENDENCIES = src/libzmq.la
```

I'll add this if everything works.

Reply:
lol, I just suggested exactly the same thing. You need this too though.

Reply:
Yup, also i tested it without the `${src_libzmq_la_LIBADD}` line and it works.